diff --git a/src/Menu.js b/src/Menu.js
index 727eac2..8f4aa99 100644
--- a/src/Menu.js
+++ b/src/Menu.js
@@ -3,7 +3,7 @@ import { Navbar, Nav, NavDropdown } from 'react-bootstrap'
 import events from './events'
 
 const Menu = _ =>
-  <Navbar bg="dark" variant="dark" expand="md">
+  <Navbar bg="dark" variant="dark" expand="md" fixed="top">
     <Navbar.Brand href="/">Finisher.Tech</Navbar.Brand>
     <Navbar.Toggle aria-controls="basic-navbar-nav" />
     <Navbar.Collapse id="basic-navbar-nav">
